subprojects/core/src/main/java/org/gradle/api/internal/StartParameterInternal.java

```diff
@@ -74,4 +91,36 @@ public boolean isSearchUpwardsWithoutDeprecationWarning() {
     public void setSearchUpwardsWithoutDeprecationWarning(boolean searchUpwards) {
         super.searchUpwards = searchUpwards;
     }
+
+    public ConfigurationCacheOption.Value getConfigurationCache() {
```

Comment:
We should follow up with a way to add options without having to add them to `StartParameter`.